internal/pkg/filesystem/file.go

```diff
@@ -118,13 +118,13 @@ func (f *RawFile) ToJsonFile() (*JsonFile, error) {
 	return file, nil
 }
 
-func (f *RawFile) ToJsonNetFile() (*JsonNetFile, error) {
+func (f *RawFile) ToJsonNetFile(vars jsonnet.VariablesValues) (*JsonNetFile, error) {
 	ast, err := jsonnet.ToAst(f.Content)
 	if err != nil {
```

Comment:
Pridany parameter `variables`, s hodnotami variables, ktore sa maju vykonat v JsonNet subore.
Pridane do vsetky metod, kde je to potrebne + doplnene testy.